app/src/test/java/com/arctouch/codechallenge/presentation/MovieDetailsPresenterTest.java

```diff
@@ -0,0 +1,36 @@
+package com.arctouch.codechallenge.presentation;
+
+import android.content.Context;
+
+import com.arctouch.codechallenge.di.InjectionMovieDetailsPresenter;
+import com.arctouch.codechallenge.movie_details.presentation.MovieDetailsContract;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import io.reactivex.schedulers.Schedulers;
+
+public class MovieDetailsPresenterTest {
+
+    @Mock
+    private Context context;
+
+    @Mock
+    private MovieDetailsContract.View view;
+
+    private MovieDetailsContract.Presenter presenter;
+
+    @Before
+    public void setup(){
+        MockitoAnnotations.initMocks(this);
+        presenter = InjectionMovieDetailsPresenter.inject(context, view, Schedulers.trampoline(), Schedulers.trampoline());
+    }
+
+    @Test
+    public void callGetMovieDetailsTest(){
+        presenter.getMovieDetails(370567);
```

Comment:
How could you test if the correct `view` methods are being called by the presenter in this unit test?

Reply:
I don’t know if I understood the question correctly, but I just mocked the view in a sense that it is passed to the injection method in order to obtain the Presenter instance and to check the call and response from this Presenter with RxJava.